requirements.txt

```diff
@@ -229,12 +229,10 @@ pexpect==4.7.0
     # via -r requirements.in
 pickleshare==0.7.5
     # via -r requirements.in
-posthoganalytics==2.0.0
+posthoganalytics==2.0.1
     # via -r requirements.in
 prometheus-client==0.14.1
     # via django-prometheus
-protobuf==3.13.0
```

Comment:
leftover from https://github.com/PostHog/posthog/pull/10932

@tiina303 friendly reminder to run pip-compile when updating dependencies :)